src/components/data_search/DatasetSearchTableConstants.tsx

```diff
@@ -402,18 +402,50 @@ export const makeDatasetTableHeader = (datasets: DatasetTerm[], selected: number
       }
     },
     {
-      label: 'DAC',
+      label: 'Data Use',
       sortable: true,
-      cellStyle: makeHeaderStyle(cellWidths.dac),
-      cellDataFn: (dataset: DatasetTerm) => ({
-        data: <OverflowTooltip place={'top'} tooltipText={dataset.dac?.dacName} id={`${dataset.datasetId}-dataset-dac`}>
-          {dataset.dac?.dacName}
-        </OverflowTooltip>,
-        value: dataset.dac?.dacName,
-        id: `${dataset.datasetId}-dac`,
-        style: makeRowStyle(cellWidths.dac),
-        label: `DAC for dataset ${dataset.datasetId}: ${dataset.dac?.dacName}`
-      })
+      cellStyle: makeHeaderStyle(cellWidths.dataUse),
+      cellDataFn: (dataset: DatasetTerm) => {
```

Comment:
This whole function reuses the Data Use logic used on the DAC Datasets table (in the file DACDatasetTableCellData.jsx) to maintain consistency in how each table's Data Use column behaves/looks. It's not as clean as the other cellDataFn's in this file so I'm open to feedback/suggestions on what to do here if this isn't the best approach

Reply:
Update - Adjusted to using helper function instead

Reply:
I think that's okay. I would rather it be different here than duplicate the logic in multiple places.